serialization/Payloads/sdoc-v1.0.json

```diff
@@ -0,0 +1,97 @@
+{
+  "namespace": "urn:spdx.dev:",
+  "created": "2022-05-02T20:28:00.000Z",
+  "creator": ["iamwillbar"],
+  "specVersion": "3.0",
+  "profile": ["core"],
+  "dataLicense": "CC0-1.0",
+  "element": [
+    {
+      "id": "null-sbom",
+      "type": {
+        "sbom": {
+          "element": ["iamwillbar", "spdx-tools-3.0.1", "project", "doc"],
+          "import": [
+            {
+              "externalId": "urn:spdx.dev:project",
+              "verifiedUsing": {
+                "hash": {
+                  "sha256": "14a657a7118a333cc1fdc6af05071a59cda067fd11130d4ee5d6d47c26e7863f"
+                }
+              },
+              "locationHint": "https://spdx.dev/projects/v1.0.json"
+            },
+            {
+              "externalId": "urn:spdx.dev:doc",
+              "verifiedUsing": {
+                "hash": {
+                  "sha256": "14a657a7118a333cc1fdc6af05071a59cda067fd11130d4ee5d6d47c26e7863f"
+                }
+              },
+              "locationHint": "https://spdx.dev/docs/v1.0.json"
+            }
+          ]
+        }
+      },
+      "profile": ["core", "software"]
+    },
+    {
+      "id": "iamwillbar",
+      "type": {
+        "person": {
+          "identifiedBy": [
+            {"email": "willbar@microsoft.com"},
+            {"account": {"authority": "github.com", "localId": "iamwillbar"}}
+          ]
+        }
+      },
+      "name": "William Bartholomew"
+    },
+    {
+      "id": "spdx-tools-3.0.1",
+      "type": {
+        "package": {
+          "packagePurpose": ["application"],
+          "downloadLocation": "https://spdx.dev/downloads/spdx-tools-3.0.1.tgz",
+          "homePage": "https://spdx.dev/tools.3.0",
+          "originator": ["project"]
+        }
+      },
+      "profile": ["core", "software"],
+      "verifiedUsing": [
+        {
+          "hash": {
+            "sha256": "14a657a7118a333cc1fdc6af05071a59cda067fd11130d4ee5d6d47c26e7863f"
+          }
+        }
+      ],
+      "externalIdentifier": [
+        {"type": "purl", "identifier": ""},
+        {"type": "cpe_2.2", "identifier": ""}
```

Comment:
I'm curious as to why you handle properties inherited from `Element` differently than the package specific properties (only the latter are grouped into the "type":"package" sub-dict). Is it because the schema is easier that way?

Reply:
Grouping the package-specific properties in the "package" property keeps them together and guarantees that there are no property name collisions without having to prepend "package" to individual properties.   I'm not sure that there would be a "purpose" property and a "downloadLocation" property elsewhere in a Package element, but for some reason the logical model qualifies one and not the other.  

Grouping them also (imo) makes element data easier to read by keeping them together, and syntax editors can collapse "type" to get a compact overview of many elements.

As you say it also makes the schema easier to write manually - the Package type defines only package properties.  It's also possible for a schema generation tool to run the inheritance chain for each element type and generate a fully-populated Package type, but that means the human-created input schema is different from the inheritance-resolved output schema.